### PR TITLE
fix: add missing EnvoyTokenAuth class properties

### PR DESCRIPTION
**Pull request by catsmanac**
auth.py had 3 property methods that were returning data from variables that are not declared at the class level. Added the class variables.

### CODE REVIEW

File: src/pyenphase/auth.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class EnvoyAuth:
 
     def __init__(self, host: str) -> None:
         """Initialize the EnvoyAuth class."""
-        pass
+        pass  # pragma: no cover
```
Comment (bdraco) [inline src/pyenphase/auth.py:21]:
```suggestion
```

**Review by bdraco — APPROVED** [review]

File: src/pyenphase/auth.py (hunk 1 above)
Comment (bdraco) [reply src/pyenphase/auth.py:21]: This looks like an unnecessary pass statement

Comment (catsmanac) [reply src/pyenphase/auth.py:21]: can very well be, it was there originally, so didn´t know if it was there for a purpose. If no use we can remove it.

Comment (bdraco) [reply src/pyenphase/auth.py:21]: Maybe auto generated. AFAICT its not needed

Comment (catsmanac) [reply src/pyenphase/auth.py]: Removed it. Tests and my Envoy still happy without it.

**Pull request merged**